tests/alluser.test.js

```diff
@@ -144,61 +144,4 @@ describe('User Controller', () => {
         done();
       });
   });
```

Comment:
#### Very Critical
So many test cases were deleted. This has drastically reduced the Coverage. Please fix this.

Reply:
The test cases deleted were redundant tests, the reason why coverage dropped was that there were some new files added when I rebased that had low coverage and this affected the overall coverage. I tried fixing it but since I wasn't the one that wrote them, there was no way I would fix the coverage without breaking something